runtime/src/ancient_append_vecs.rs

```diff
@@ -372,6 +405,16 @@ struct AccountsToCombine<'a> {
     target_slots: Vec<Slot>,
 }
 
+#[allow(dead_code)]
+#[derive(Default)]
+/// intended contents of a packed ancient storage
```

Comment:
nit: typically annotations go below comments on types, fns, etc
```suggestion
/// intended contents of a packed ancient storage
#[allow(dead_code)]
#[derive(Default)]
```